static/src/javascripts/projects/commercial/modules/sticky-mpu.js

```diff
@@ -64,7 +64,8 @@ const stickyMpu = (adSlot: HTMLElement) => {
     }
 
     const referenceElement: ?HTMLElement = document.querySelector(
-        '.js-article__body,.js-liveblog-body-content'
+        '.js-article__body:not([style*="display: none;"]), ' +
+        '.js-liveblog-body-content:not([style*="display: none;"])'
```

Comment:
Might be worth making this into a array of css selectors joined by a comma and space, as below:

```suggestion
        ['.js-article__body:not([style*="display: none;"])',
        '.js-liveblog-body-content:not([style*="display: none;"])'].join(', ')
```

It might be clearer for future extensions.